contrib/test/txn-fixtures/program-tests.list

```diff
@@ -38,3 +38,4 @@ dump/test-vectors/txn/fixtures/programs/d5e2332f53032ce4428b0b55e3a97c080c1db63e
 dump/test-vectors/txn/fixtures/programs/deff84b60ea3c6284e621b4229f402c4274e4968_1446008.fix
 dump/test-vectors/txn/fixtures/programs/eaed07290ea8722cb900e1e648ca5209fa35ef6c_3861880.fix
 dump/test-vectors/txn/fixtures/programs/ed4565e33252bca4dc606bdce4aa48a650e75048_1366919.fix
+dump/test-vectors/txn/fixtures/programs/2e6cbb44ce7014cd45620872b31af391476032a2_1684472.fix
```

Comment:
nit: can you put this in alphabetical order? it makes the diff much smaller in the future when running `find dump/test-vectors/txn/fixtures/programs | sort > program-tests.list`